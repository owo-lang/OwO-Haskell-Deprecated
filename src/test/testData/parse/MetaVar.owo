module A where
  x = _
  z = id Unit _A
