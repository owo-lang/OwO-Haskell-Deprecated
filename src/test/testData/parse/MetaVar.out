File type: owo
Top module name: .A
 implementation of function x (17 2 3) (18 2 4)
  no pragmas
  clause for x (17 2 3) (18 2 4)
  patterns matching parsed as expression
   named reference x (17 2 3) (18 2 4)
  function body
   named metavar _ (21 2 7) (22 2 8)
  no with abstraction
  no where clause
 implementation of function z (25 3 3) (26 3 4)
  no pragmas
  clause for z (25 3 3) (26 3 4)
  patterns matching parsed as expression
   named reference z (25 3 3) (26 3 4)
  function body
   function application
   function being applied
    function application
    function being applied
     named reference id (29 3 7) (31 3 9)
    value applied to the function
     named reference Unit (32 3 10) (36 3 14)
   value applied to the function
    named metavar _A (37 3 15) (39 3 17)
  no with abstraction
  no where clause
